### ISSUE TITLE
Sorting issue with negative values on site

### ISSUE THREAD

**Issue by TrystanOakley** [sstangl/openpowerlifting#282]
If you sort squat, bench, or deadlift by lowest to highest there are negative numbers. Not sure why.

**Comment by sstangl** [comment]
We allow negative numbers in the BestLiftKg column. Some federations (mostly USPA) use that to report failed attempts.

You are right that they shouldn't appear in the sort, though. They should basically be invisible. But it's useful to keep them around for... later projects...

**Comment by TrystanOakley** [comment]
Wouldn't a failed attempt indicate that they just...disqualified?

**Comment by sstangl** [comment]
Yes, but we track explicitly why they were disqualified. The whole "Event" column is basically just for that purpose also (also to catch errors in testing). It will be used for "something" later.
